tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
+    "allowJs": true,
     "esModuleInterop": true,
-    "isolatedModules": true,
```

Comment:
@michaelsbradleyjr I needed to remove `isolatedModules` here because we have some files inside `utils` that don't expose a module.